### ISSUE TITLE
Enhance Shop/Upgrade tree UI

### ISSUE THREAD

**Issue by Monkey-bored** [Ido-Barnea/Chess-But-Better#243]
### Requested Changes
Currently the shop/upgrades tree items just seem to float somewhere, instead add a visual border around the the shop/upgrades tree and text at the top of the area inside the border that says "Shop" or "Upgrades tree" accordingly.

![image](https://github.com/Ido-Barnea/Chess-But-Better/assets/62211288/12912f41-1919-41db-8ddc-95c3e6c43d94)
Something like this

**Comment by Ido-Barnea** [comment]
@Monkey-bored May I also suggest this change?
![image](https://github.com/Ido-Barnea/Chess-But-Better/assets/30974296/92846dd7-1bff-4ed9-ad61-fd5723fb7953)

**Comment by Monkey-bored** [comment]
> @Monkey-bored May I also suggest this change?
> ![image](https://github.com/Ido-Barnea/Chess-But-Better/assets/30974296/92846dd7-1bff-4ed9-ad61-fd5723fb7953)
> 

It kind of hides the items svg behind the price

**Comment by Ido-Barnea** [comment]
> > @Monkey-bored May I also suggest this change?
> > ![image](https://private-user-images.githubusercontent.com/30974296/320215769-92846dd7-1bff-4ed9-ad61-fd5723fb7953.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI0MTk4MjksIm5iZiI6MTcxMjQxOTUyOSwicGF0aCI6Ii8zMDk3NDI5Ni8zMjAyMTU3NjktOTI4NDZkZDctMWJmZi00ZWQ5LWFkNjEtZmQ1NzIzZmI3OTUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA2VDE2MDUyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1ZGI3NTc3YjFlODU5N2M3NDM4NjFlNGRiZDNlM2M3MGUxOTc4YjA5NmIxMmZiNjcwZTk2OGRmMWJmMTExOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tq56gTHpNRn9-Hdr8qLtNP7RlWpFzQkzSSkyR4gu-L4)
> 
> It kind of hides the items svg behind the price

Yes well I'm not an MS Paint God or anything but my intention was that the items would be higher (guess I could have painted it correctly but 🤓)

**Comment by Ido-Barnea** [comment]
> @Monkey-bored May I also suggest this change? ![image](https://private-user-images.githubusercontent.com/30974296/320215769-92846dd7-1bff-4ed9-ad61-fd5723fb7953.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM4NjEyNDgsIm5iZiI6MTcxMzg2MDk0OCwicGF0aCI6Ii8zMDk3NDI5Ni8zMjAyMTU3NjktOTI4NDZkZDctMWJmZi00ZWQ5LWFkNjEtZmQ1NzIzZmI3OTUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIzVDA4MjkwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4OTQwNDk5OTVkMzhkYzQyYzkwNzFmMzNlOGEzMjUyOTRiYzc0MWY3NWQ2MzJkZmNjYzBmMzU4NTY5ZDZhOWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LaSSRxNdiLKZ-hmxJUk9vQLHWxILWz9EA2iIdF_eigk)

@Monkey-bored Please only add the border and title as my previous suggestion will be solved in #250

**Comment by Ido-Barnea** [comment]
@Monkey-bored is there an update on this?
